modules/ztp-deleting-node-using-siteconfig.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ztp-advanced-install-ztp.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-deleting-node-siteconfig_{context}"]
+= Deleting a node using the SiteConfig CR
+
+You can use a SiteConfig custom resource (CR) to delete a node and then reprovision it. 
+This is more efficient than manually deleting the node.
+
+.Prerequisites
+
+* You configured the hub cluster for generating the required installation and policy CRs.
+
+* You created a Git repository where you manage your custom site configuration data. The repository must be accessible from the hub cluster and be defined as a source repository for the Argo CD application.
+
+
+.Procedure
+
+. Update the  `SiteConfig` CR to include the `bmac.agent-install.openshift.io/remove-agent-and-node-on-delete=true` annotation, for example:
++
+[source,yaml]
+----
+nodes:
+- hostname: node6
+  role: "worker"
+  crAnnotations:
+    add:
+      BareMetalHost:
+        bmac.agent-install.openshift.io/remove-agent-and-node-on-delete: true
+
+----
+
+. Suppress the generation of the `BMH` CR via SiteConfig. Update the `SiteConfig` CR to include the `crSuppression.BareMetalHost`, for example:
++
+[source,yaml]
+----
+nodes:
+- hostname: node6
+  role: "worker"
+  crSuppression:
+    - BareMetalHost
+----
+
+. Push the changes to the git repo and wait for deprovisioning to start. 
+The status of the `BMH` CR should change to "deprovisioning". Wait for the `BMH` to finish deprovisioning, and to be fully deleted.
+
+.Verification
+
+. Verify that the `BMH` and `Agent` CRs for the worker node have been deleted from the hub cluster by running the following commands:
++
+[source,terminal]
+----
+$ oc get bmh -n <cluster-ns>
+$ oc get agent -n <cluster-ns>
+----
+
+. Verify that the node record has been deleted from the spoke cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+[NOTE]
+====
+When working with secrets, deleting a secret too soon may cause an issue as ArgoCD needs the secret to complete the resync following deletion. 
+Do not delete the secret until after node cleanup when the current ArgoCD sync has completed.
+====
+
+.Reprovisioning
```

Comment:
Next steps??

Reply:
@agil antony
Thanks for the review. I'm not sure what you mean here

Reply:
+1 renaming or moving this. The title of this module is "Deleting a node by using the SiteConfig CR" and walks through the steps to delete. The only mention of reprovisioning is in the intro paragraph. As of now, naming this "reprovisoning" not only seems out of place, but I couldn't find anywhere else in the docs where we do this. It also is not listed in the [mod docs guide](https://redhat-documentation.github.io/modular-docs/#verification) as an option for procedure modules either.

Since ["Next steps"](https://redhat-documentation.github.io/modular-docs/#next-steps) in the mod docs guide says not to provide a second list of instructions which I feel like this step kind of starts to. I would suggest maybe making another module for reprovisioning because while you could move it up to the last step before the verification steps, it still seems out of place there too.

Reply:
Thanks, @agantony  After some discussion we've gone with your suggestion of "Next Steps".